src/test/java/com/meilisearch/integration/SearchTest.java

```diff
@@ -315,5 +315,58 @@ public void testSearchMatches() throws Exception {
 		assertEquals(375, res_gson.hits[0].getMatchesInfo().get("overview").get(2).start);
 		assertEquals(3, res_gson.hits[0].getMatchesInfo().get("overview").get(2).length);
 	}
-	
+	/**
+	 * Test place holder search
+	 */
+	@Test
+	public void testPlaceHolder() throws Exception {
+		String indexUid = "placeHolder";
+		Index index = client.index(indexUid);
+		GsonJsonHandler jsonGson = new GsonJsonHandler();
+
+		TestData<Movie> testData = this.getTestData(MOVIES_INDEX, Movie.class);
+		UpdateStatus updateInfo = jsonGson.decode(
+			index.addDocuments(testData.getRaw()),
+			UpdateStatus.class
+		);
+
+		index.waitForPendingUpdate(updateInfo.getUpdateId());
+
+		Results res_gson = jsonGson.decode(
+			index.search(""),
+			Results.class
+		);
+		assertEquals(index.search(""), index.search(new SearchRequest(null)));
```

Comment:
I would suggest storing the results of `index.search("")` in a Sting variable and then use it in both places (L336 and L339). This won't probably add a noticeable difference now, but seems to me to be a more consistent way of knowing the values are the same always, and also avoid 1 call to the server :)

Reply:
Good catch @eskombro! I didn't notice I was actually making one unnecessary call to the server. I'll implement the changes.